README.md

```diff
@@ -57,6 +58,9 @@ module.exports.handler = serverless(app, {
       .then(() => {
         res.foo = 'bar';
       });
+  },
+  transform: (event, context) => (req, res) => {
+    res.locals.foo = event.stageVariables.foo;
```

Comment:
Is res.locals a standard thing on http.IncomingMessage? Because this is being called before a web framework, meaning that if - for example - express or koa build their own request objects (which I'm pretty sure they do) these locals will be missing.. I realize this is just an example but it could be confusing to people.